charts/alluxio/values.yaml

```diff
@@ -80,6 +80,7 @@ master:
 #    label2: value2
 
   annotations:
+    sidecar.istio.io/inject: "false"
```

Comment:
I think the default value could be overwrite by alluxio  controller. Instead, I suggest inserting this code into the line 46.

https://github.com/fluid-cloudnative/fluid/blob/c8dbd7eb718aef0e2b8e11b613afdb246860ee33/charts/alluxio/templates/master/statefulset.yaml#L45-L49